### PR TITLE
Correct `Options.columns` to allow a list of key/header objects

### PR DESCRIPTION
**Pull request by yonilerner**
Addresses https://github.com/adaltas/node-csv-stringify/issues/73

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -20,6 +20,11 @@ declare namespace stringify {
     }
 
     type Input = any[] | PlainObject<any>
+    
+    interface ColumnOptions {
```
Comment (nene) [inline lib/index.d.ts]: I think this interface would be better named `ColumnOption` (in singular).
```suggestion
    interface ColumnOption {
```

File: lib/index.d.ts [hunk 2]
```diff
@@ -30,7 +35,7 @@ declare namespace stringify {
          * can refer to nested properties of the input JSON
          * see the "header" option on how to print columns names on the first line
          */
-        columns?: string[] | PlainObject<string>
+        columns?: string[] | PlainObject<string> | ColumnOptions[]
```
Comment (nene) [inline lib/index.d.ts]: And then in here you'd have the plural: an array of column options.

```suggestion
        columns?: string[] | PlainObject<string> | ColumnOption[]
```

**Comment by yonilerner** [comment]
@wdavidw @nene Suggestions applied

**Comment by wdavidw** [comment]
Thanks

**Comment by wdavidw** [comment]
@yonilerner could you rebase the branch yonilerner-patch-1 on master ?

**Comment by yonilerner** [comment]
@wdavidw Done

**Comment by wdavidw** [comment]
Thank you!

**Pull request merged**